src/docs/product/sentry-basics/search/index.mdx

```diff
@@ -200,33 +200,33 @@ Several common uses for tags include:
 
 ## Premade Searches
 
-Premade searches are common search terms that we think you're likely to use. The premade searches will appear in the order of which you've most recently used them.
+Premade searches are common search terms that we think you're likely to use. The premade searches are listed under "Recommended Searches" in the "Saved Searches" dropdown, and they're listed in order of when you've most recently used them.
 
-![](search-premade-dropdown.png)
+![Open Saved Searches dropdown on the Issues page.](search-premade-dropdown.png)
 
 ## Pinned Searches
 
-You can pin a search, and it will become the default view you see on your Issues view. The pinned search is only visible to you and is relevant across your projects.
+You can pin a search, and it will become the default view you see on the **Issues** page. The pinned search is only visible to you and is relevant across your projects.
 
 1. Type a search into the search bar.
 
-   ![](search-pinning-step-one.png)
+   ![The search bar of the Issues page with two search terms.](search-pinning-step-one.png)
 
 2. Click the pin icon next to that search.
 
-   ![](search-pinning-step-two.png)
+   ![Cursor mousing over the pin icon and tool-tip displayed.](search-pinning-step-two.png)
 
-3. Once pinned, Sentry will add the search to the Saved Search Dropdown. The search label in the text will read: My Pinned Search.
+3. Once pinned, Sentry will add the search to the "Saved Searches" dropdown. The search label in the text will read, "My Pinned Search".
 
-   ![](search-pinning-step-three.png)
+   ![Pinned search in Saved Searches dropdown.](search-pinning-step-three.png)
 
 ### Changing a Pinned Search
 
 To change your pinned search, do the following:
 
 1. Select your pinned search. Un-click the pin icon. Your default search will return to `is:unresolved`.
 
-2. Do another search. Click the pin icon. The query listed as 'My Pinned Search' will now be the new pinned query, instead of the original one.
+2. Do another search. Click the pin icon. The query listed as "My Pinned Search" will now be the new pinned query, instead of the original one.
```

Comment:
```suggestion
2. Run another search. Click the pin icon. The query listed as "My Pinned Search" will now be the new pinned query, instead of the original one.
```